tests/test_web/test_broker_error.py

```diff
@@ -0,0 +1,218 @@
+from unittest import mock
+
+from kombu.exceptions import OperationalError
+
+from iib.web.models import Request, RequestStateMapping, RequestState, RequestAdd, RequestRm
+
+
+def assert_testing(rv, mock_smfsc, db):
+    response_json = {'error': "The scheduling of the build request with ID 1 failed"}
```

Comment:
Could you use single quotes please.

Reply:
I guess PyCharm is set to modify it automatically, so I did not notice that at first. But when you pointed out it looks like all recently edited strings were transformed to double quote. 
Fixed.